ScratchWikiSkin.skin.php

```diff
@@ -49,15 +49,21 @@ class SkinScratchWikiSkin extends SkinTemplate {
 		$template = 'ScratchWikiSkinTemplate', $useHeadElement = true;
 
 	/**
-	 * Add CSS via ResourceLoader
+	 * This function adds JavaScript via ResourceLoader
 	 *
-	 * @param $out OutputPage
+	 * Use this function if your skin has a JS file(s).
+	 * Otherwise you won't need this function and you can safely delete it.
+	 *
+	 * @param OutputPage $out
 	 */
-	public function setupSkinUserCss( OutputPage $out ) {
-		parent::setupSkinUserCss( $out );
+
+	public function initPage( OutputPage $out ) {
+		parent::initPage( $out );
+		$out->addModules('skins.scratchwikiskin2.js');
 		$out->addModuleStyles( [
 			'mediawiki.skinning.interface', 'skins.scratchwikiskin2'
-		] );
+			/* 'skins.foobar' is the name you used in your skin.json file */
+		 ] );
```

Comment:
...why, why is there an indentation change?
```suggestion
		] );
```